chrome/content/zotero/xpcom/data/dataObjects.js

```diff
@@ -74,6 +72,9 @@ Zotero.defineProperty(Zotero.DataObjects.prototype, 'primaryFields', {
 	get: function () Object.keys(this._primaryDataSQLParts)
 }, {lazy: true});
 
+Zotero.defineProperty(Zotero.DataObjects.prototype, 'primaryDataSQLFrom', {
+	get: function() " " + this._primaryDataSQLFrom + " " + this._primaryDataSQLWhere
+}, {lateInit: true});
```

Comment:
This and the change above are actually no longer relevant. Some hacky code relied on this previously.